content/tutorials/manage-the-rippled-server/installation/update-rippled-automatically-on-centos-rhel.md

```diff
@@ -0,0 +1,25 @@
+# Update Automatically on CentOS/Red Hat
+
+On CentOS and Red Hat Enterprise Linux, you can set up `rippled` to automatically upgrade to the latest version with a one-time `cron` configuration. Ripple recommends enabling automatic updates if possible.
+
+These instructions assume you have already [installed `rippled` from the `yum` repository](install-rippled-on-centos-rhel-with-yum.html).
+
+To set up automatic updates, complete the following steps:
+
+1. Check that `/opt/ripple/bin/update-rippled.sh` exists. If it does not, [update manually](update-rippled-manually-on-centos-rhel.html).
+
+2. Install `crond`:
+
+        $ sudo yum install cronie
```

Comment:
is `cronie` a typo ?

Reply:
No, cronie is the name of a specific implementation of the cron daemon and also [the name of the package in the Red Hat repos](https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/7/html/system_administrators_guide/ch-automating_system_tasks#s1-Scheduling_a_Recurring_Job_Using_Cron).